### PR TITLE
[Doc]--Update BookKeeper config

### LINKED ISSUE #7481: Default value and in-line doc for bookkeeper config: openLedgerRereplicationGracePeriod is incorrect
**Describe the bug**
Default bookkeeper conf that ships with apache pulsar indicates that the unit of `openLedgerRereplicationGracePeriod` is in seconds, but official bookkeeper doc(and code) mentions that the unit is in milliseconds. Since default is 30seconds, this will be interpreted by BK as 30ms because of incorrect configuration by pulsar

**To Reproduce**
1. Go to https://bookkeeper.apache.org/docs/4.10.0/reference/config/ and read documentation for `openLedgerRereplicationGracePeriod`. It clearly mentions that the period is in milliseconds and default is `30000`. This can be further confirmed by seeing usage of grace period in https://github.com/apache/bookkeeper/blob/master/bookkeeper-server/src/main/java/org/apache/bookkeeper/replication/ReplicationWorker.java where it is treated as milliseconds
1. Download apache pulsar release (checked in 2.5.2 and 2.6.0) or go here: https://github.com/apache/pulsar/blob/master/conf/bookkeeper.conf
1. Observe that `openLedgerRereplicationGracePeriod=30` is configured and comments above this mention that the period is in seconds
1. Pulsar will end up configuring grace period as 30ms which is not the expected behaviour (30s is expected)

**Expected behavior**
Pulsar should configure default as `30000` instead of `30` and bookkeeper.conf should mention that the unit of this period is milliseconds instead of seconds.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Debian 9

**Additional context**

### PR DESCRIPTION
**Pull request by Huanli-Meng**
Fixes #7481 

### Motivation
Default bookkeeper conf that ships with apache pulsar indicates that the unit of `openLedgerRereplicationGracePeriod` is in seconds, but official bookkeeper doc(and code) mentions that the unit is in milliseconds. 

Pulsar should configure default as 30000 instead of 30 and bookkeeper.conf should mention that the unit of this period is milliseconds instead of seconds.

### Modifications
1. Update the bookkeep.config file
2. Update the following Pulsar docs for releases 2.5.2, 2.6.0, 2.6.1, and master

Reference > Pulsar Config > BookKeeper

### CODE REVIEW

**Comment by Huanli-Meng** [comment]
@jiazhai , please help review the PR. Thanks.

File: conf/bookkeeper.conf [hunk 1]
```diff
@@ -227,9 +227,9 @@ auditorPeriodicBookieCheckInterval=86400
 rereplicationEntryBatchSize=100
 
 # Auto-replication
-# The grace period, in seconds, that the replication worker waits before fencing and
+# The grace period, in milliseconds, that the replication worker waits before fencing and
 # replicating a ledger fragment that's still being written to upon bookie failure.
-openLedgerRereplicationGracePeriod=30
+openLedgerRereplicationGracePeriod=3000
```
[the remark this answers is not in the document]
Comment (Huanli-Meng) [reply conf/bookkeeper.conf]: > turn into 30000 instead of 3000

@jiazhai , doc is updated accordingly. Please help double check. Thanks.

**Review by jiazhai — APPROVED** [review]

**Pull request merged**